docs/hardware/1-home.md

```diff
@@ -5,37 +5,56 @@ hide_title: false
 slug: /hardware
 ---
 
-The Hardware section includes supported hardware for the Golioth platform. Includes hardware description, how to get started using that hardware on the Golioth platform, and how to apply other Golioth examples to that hardware.
+The Hardware section includes a catalog of supported boards for the Golioth
+platform. Additionally, there are quickstart guides and recommended boards for
+common MCUs, such as the ESP32 and nRF91.
 
-## Classifications of Hardware
+## Board Support Tiers
 
-### Verified + Quickstart Boards
+Golioth has three levels of board support.
 
-These are Verified Boards (see below) that also have a quickstart guide written for the board.
+### Continuously Verified Boards
+
+A continuously verified board:
+
+* is highly recommended for new users that want to try out Golioth.
+* is tested on every release of the [Golioth Zephyr SDK](https://github.com/golioth/golioth-zephyr-sdk).
+* is regularly tested and used by the Golioth development team.
+* has first class support and maintenance from Golioth. If you encounter a problem
+    with one of these boards, you can reach out to us on [Discord](https://golioth.io/discord) or
+    [file a bug report](https://github.com/golioth/golioth-zephyr-sdk/issues),
+    and we will address it quickly.
+
+The set of boards in this category covers commonly used
+connectivity options including WiFi, cellular, and Thread.
+Additionally, these boards cover common MCUs, such as the ESP32 and nRF91.
 
 ### Verified Boards
 
-These are boards or devices that members of the Golioth team have used and confirm work well using the Golioth platform. There will be additional support, such as overlay files in the various firmware samples, and regular testing that goes along with the "verified status". Verified boards do not yet have quickstart guides associated with them.
+A verified board:
+
+* is tested and confirmed to work with Golioth.
+* is tested less frequently than continously verified boards. This might mean it
+    was tested on an older version of the Golioth Zephyr SDK, or that
+    it has been a while since it was last tested.
```

Comment:
```suggestion
* is tested less frequently than continuously verified boards. This means it
    was tested on an older version of the Golioth Zephyr SDK, but may not
    have been tested on the most recent commits.
```

This is a typo correction and a suggestion for rewording the second half of this bullet point.